frontend/src/store/todos-context.spec.tsx

```diff
@@ -1,54 +1,32 @@
 import React from 'react';
-import { fireEvent, render, screen } from '@testing-library/react';
 import TodosContextProvider, { TodosContext } from './todos-context';
+import Enzyme, { mount } from 'enzyme';
+import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
 
 describe('TodoProvider', ()=>
 {
-
-	test('listing the todos and checks if the array length is not 0', ()=>
+	Enzyme.configure({ adapter: new Adapter() });
+	it('sets status', () =>
 	{
+		const TestComponent : React.FC = () =>
+		{
+			const { addTodo, items } = React.useContext(TodosContext);
 
-		render(<TodosContextProvider>
-			<TodosContext.Consumer>
-				{
-					value => <span>Is length: {value.fetchedItems}</span>
-				}
-			</TodosContext.Consumer>
-		</TodosContextProvider>);
-	expect('Is length').not.toHaveLength(0);
-	});
-
-	test('checking the length on the exact number of todos', ()=>
-	{
+			return <React.Fragment>
+				<div data-testing = 'value'>{items.map(item=>item.title)}</div>
+				<button onClick={() => addTodo('Test Todo!')}>Add Todo</button>
+			</React.Fragment>;
+		};
+		const wrapper : Enzyme.ReactWrapper<any> = mount(
+			<TodosContextProvider>
+				<TestComponent />
+			</TodosContextProvider>
+		);
 
-		render(<TodosContextProvider>
-			<TodosContext.Consumer>
-				{
-					value => <span>Is length: {value.fetchedItems}</span>
-				}
-			</TodosContext.Consumer>
-		</TodosContextProvider>);
-	expect(('is length')).toHaveLength(9);
-	});
+		expect(wrapper.find('[data-testing="value"]').text()).toBeFalsy();
 
-	test('checking add todo func', () =>
-	{
+		wrapper.find('button').simulate('click');
 
-		render(<TodosContextProvider>
-			<TodosContext.Consumer>
-				{
-					value => <>
-						<span>Is sending: {value.addTodo}</span>
-						<button onClick={() =>
-						{
-							value.addTodo('My Todo from Jest');
-						}
-						}>Add Todo</button>
-					</>
-				}
-			</TodosContext.Consumer>
-		</TodosContextProvider>);
-	fireEvent.click(screen.getByText('Add Todo'));
-	expect(('Is sending: true')).toBeTruthy();
+		expect(wrapper.find('[data-testing="value"]').text()).toBeTruthy();
```

Comment:
value is not a proper name... I have to look it up in the code to know what this elemen is